__tests__/database/migration/file/index.test.ts

```diff
@@ -7,6 +7,6 @@ describe('TESTING all migration file', () => {
     // Execution
 
     // Assertion
-    expect(allMigrations.length).toEqual(7);
+    expect(allMigrations.length).toEqual(8);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The test is updated to expect 8 migrations instead of the previous 7. This change is consistent with the PR summary, which mentions a new migration version 7 being added. Therefore, this change seems appropriate.
```diff
-    expect(allMigrations.length).toEqual(7);
+    expect(allMigrations.length).toEqual(8);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->